MachObfuscator/SymbolsCollecting/ObfuscationSymbols+Building.swift

```diff
@@ -48,7 +48,11 @@ extension ObfuscationSymbols {
             LOGGER.warn("Some selectors specified on blacklist were not found: \(notFoundBlacklistedSelectors)")
         }
 
-        let blacklistSelectors = (Array(blackListGetters) + Array(blacklistSetters) + Mach.libobjcSelectors + objcOptions.selectorsBlacklist + blacklistedSelectorsByRegex).uniq
```

Comment:
Originally it took more than 2s to compile.

Reply:
I observed that in more complex cases, also ones involing optionals, type inference seems to be very slow, but in the end it gets correct result unless it is terminated after 5 or 10 seconds...

Reply:
Maybe we should work on some code-compilation optimiser :D